datafusion/core/tests/sqllogictests/test_files/ddl.slt

```diff
@@ -45,9 +45,9 @@ query II rowsort
 select * from users;
 ----
 1 2
+11 20
```

Comment:
Why this change?

Reply:
For rowsort, runner will only sort actual results now, which means the result in the test cases should be sorted.